index.html

```diff
@@ -434,20 +435,6 @@
       document.title = this.defaultTitle_;
     }
   },
-  updateFrameURL: function(url) {
-    if (!/\.html$/.test(url)) {
-      // When loading a plain string, e.g. "galaxy", Chrome will attempt to
-      // load the http version even if the site is loaded on https. Specifying
-      // the index page seems to fix this.
-      url += '/index.html';
-    }
```

Comment:
After https://github.com/web-animations/web-animations-demos/pull/79 we no longer need this hack.